pageserver/src/tenant/storage_layer/layer.rs

```diff
@@ -1782,13 +1782,12 @@ impl ResidentLayer {
 
     /// FIXME: truncate is bad name because we are not truncating anything, but copying the
     /// filtered parts.
-    #[cfg(test)]
-    pub(super) async fn copy_delta_prefix(
+    pub(crate) async fn copy_delta_prefix(
         &self,
         writer: &mut super::delta_layer::DeltaLayerWriter,
         truncate_at: Lsn,
         ctx: &RequestContext,
-    ) -> anyhow::Result<()> {
+    ) -> anyhow::Result<usize> {
```

Comment:
Please adjust the method comment to explain what the `usize` describes that we return here.

Reply:
Added cf38f6ed2